project-base/storefront/components/Blocks/Product/Filter/FilterSelectedParameters.tsx

```diff
@@ -109,25 +110,16 @@ export const FilterSelectedParameters: FC<FilterSelectedParametersProps> = ({ fi
                             <SelectedParametersName>{selectedParameterOptions?.name}:</SelectedParametersName>
                             {isSliderParameter ? (
                                 <SelectedParametersListItem key={selectedParameterOptions.uuid}>
-                                    {selectedParameterOptions.minimalValue && (
-                                        <>
-                                            <span>{t('from')}&nbsp;</span>
-                                            {selectedParameterOptions.minimalValue}
-                                            {selectedParameterOptions.unit?.name !== undefined
-                                                ? `\xa0${selectedParameterOptions.unit.name}`
-                                                : ''}
-                                            {selectedParameterOptions.maximalValue && ' '}
-                                        </>
-                                    )}
-                                    {selectedParameterOptions.maximalValue && (
-                                        <>
-                                            <span>{t('to')}&nbsp;</span>
-                                            {selectedParameterOptions.maximalValue}
-                                            {selectedParameterOptions.unit?.name !== undefined
-                                                ? `\xa0${selectedParameterOptions.unit.name}`
-                                                : ''}
-                                        </>
-                                    )}
+                                    <span>{t('from')}&nbsp;</span>
+                                    {selectedParameter.minimalValue || selectedParameterOptions.minimalValue}
+                                    {!!selectedParameterOptions.unit?.name &&
+                                        `\xa0${selectedParameterOptions.unit.name}`}
+                                    {selectedParameterOptions.maximalValue && ' '}
+
+                                    <span>{t('to')}&nbsp;</span>
+                                    {selectedParameter.maximalValue || selectedParameterOptions.maximalValue}
+                                    {selectedParameterOptions.unit?.name && `\xa0${selectedParameterOptions.unit.name}`}
+
```

Comment:
I am not sure, if I follow. I can see `selectedParameterOptions.maximalValue` twice. Does that mean it can get rendered twice?